index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+    <div>
+        <header class="header">
+            <nav class="navbar">
+                <div class="mylogo"><h5>CB</h5></div>
+                <div><a href="#"><img class="menu resize" src="./Images/842370_multimedia_512x512.png" alt=""></a></div>
+            </nav>
+        </header>
+    </div>
+    <div class="content">
+        <div class="owner greeting">I'm Christian <br>Nice to <br> meet you!</div>
+        <div class="owner description">I’m a software developer! I can help you build a product , feature or website
+            Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</div>
```

Comment:
- [ ] The paragraph (e.g., "I'm a software ...") should be a `p` tag. This is to ensure to enhance the accessibility of your application.

Reply:
- Nice job with fixing that part.